Lesson33/Gokhan/src/App.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { List } from './components/List';
+import { Navbar } from './components/Navbar';
+import { Account } from './components/Account';
+import { PerformanceState } from './components/PerformanceState';
+import { LogIn } from './components/LogIn';
+import { UserProvider, useUserContext } from './components/Context/UserContext';
+import './App.css';
+
+const initialTodos = [
+	{ id: 1, text: 'Do not give up', completed: false },
+	{ id: 2, text: 'Keep Dreaming', completed: true },
+	{ id: 3, text: 'Follow Your Dreams', completed: false },
+	{ id: 4, text: 'Work Hard', completed: false },
+	{ id: 5, text: 'Think Possitive', completed: true },
+	{ id: 6, text: 'Finish Course', completed: false },
+];
+
+export const App = () => {
+	const [todos, setTodos] = useState(initialTodos);
+
+	return (
+		<UserProvider>
```

Comment:
If you move UserProvider to the index.js and wrap <App> component there inside it, you'll ba able to access the User context directly in this component.